src/styles/Navbar.css

```diff
@@ -0,0 +1,106 @@
+@import url('https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@100;400&display=swap');
```

Comment:
Can we remove this file? If we haven't ported all CSS to CSS-in-js let's do that and remove.